### PR TITLE
create issue on failure

### LINKED ISSUE #122: Trigger CI here after tea/cli is released
So we can verify the installer and action work with the latest release.

Potentially getting us ahead of any breakage.

### PR DESCRIPTION
**Pull request by jhheider**
fired from https://github.com/teaxyz/cli/pull/334
resolves #122 

If you like saucier templates, it's a simple `.md` file.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (mxcl) [inline .github/ISSUE_TEMPLATE.md:1]: I believe this becomes the default template for *all* issues tho. Even if a user creates an issue.

Comment (jhheider) [reply .github/ISSUE_TEMPLATE.md:1]: Yes, it seems to (I saw that in pantry core). I'll switch to a named non-default template if possible.

File: .github/workflows/ci-pre-reqs.yml [hunk 2]
```diff
@@ -20,16 +20,16 @@ jobs:
...
-        # - ubuntu:lunar     glibc too old
+        # - ubuntu:lunar       glibc too old
         # - ubuntu:trusty
-        # - ubuntu:xenial     glibc too old
+        # - ubuntu:xenial      glibc too old
         - fedora:latest
-        # - nixos/nix:latest  can’t exec
-        # - alpine:latest     tea binary doesn’t actually end up on the system (wtf)
+        # - nixos/nix:latest   can’t exec
+        - alpine:latest      # tea binary doesn’t actually end up on the system (wtf)
```
Comment (mxcl) [inline .github/workflows/ci-pre-reqs.yml:32]: causing CI failure here

**Comment by jhheider** [comment]
accidentally rebased in your +alpine commit.

**Comment by jhheider** [comment]
> Sorry to say this now, but can we separate this into a separate job that invokes CI and then notifies called `smoke-test.yml` or something like that.

np

**Pull request merged**